device-types/Cisco/C9500-16X-A.yaml

```diff
@@ -0,0 +1,59 @@
+---
+manufacturer: Cisco
+model: Catalyst 9500-16X-A
+slug: cisco-c9500-16x-a
+part_number: C9500-16X-A
```

Comment:
"-A" is just a license identifier for ordering.  The actual model number is C9500-16X.

AP's are different with the "-A" or "-I", etc as those are regional locks and the AP actually reports the specific model number for the region.

```
NAME: "Switch 1", DESCR: "C9500-16X"
PID: C9500-16X         , VID: V02  , SN:  <REDACTED>
```